### DISCUSSION TITLE
configure: error: OpenSSL version test program failed.

### DISCUSSION THREAD

**Discussion by nagu1137** [openssl/openssl#24891] [Q&A]
Im using "Ubuntu 18.04.6 LTS"
Trying to compile the openssh 9.8 and getting the following error

checking OpenSSL header version... failed
configure: error: OpenSSL version test program failed.

[[outoutssh9.8.txt](https://github.com/user-attachments/files/16232337/outoutssh9.8.txt).log](https://github.com/user-attachments/files/16232326/config.log)

i installed 3.0.9 in the /mnt/lfs/usr using the following commands.

./config --prefix==/mnt/lfs/usr --openssldir=/mnt/lfs/etc/ssl --libdir=/mnt/lfs/lib shared zlib-dynamic
make
sudo make install

**Comment by nagu1137** [comment]
Im able to compile openssh 7.9  with out any isuses by installing the 1.0.2 U with same method as above.

**Comment by nhorman** [comment]
you'll have to contact the openssh folks on this one.  Its a separate project and they will be best able to tell you how they discover the openssl location
